fdbbackup/backup.actor.cpp

```diff
@@ -2390,21 +2398,66 @@ ACTOR Future<Void> abortDBMove(Database database, Key prefix, MovementLocation l
 ACTOR Future<Void> abortDBMove(Optional<Database> src,
                                Optional<Database> dest,
                                Optional<Key> sourcePrefix,
-                               Optional<Key> destinationPrefix) {
+                               Optional<Key> destinationPrefix,
+                               AbortState abortInstruction) {
 	ASSERT(src.present() || dest.present());
 
+	state bool destinationAbortError = true;
 	try {
-		if (src.present() && sourcePrefix.present()) {
-			wait(abortDBMove(src.get(), sourcePrefix.get(), MovementLocation::SOURCE));
+		state AbortState destAbortResult = AbortState::UNKNOWN;
+		if (dest.present()) {
+			ASSERT(destinationPrefix.present());
+			AbortState tempDestAbortResult =
+			    wait(abortDBMove(dest.get(), destinationPrefix.get(), MovementLocation::DEST, abortInstruction));
+			destAbortResult = tempDestAbortResult;
+			if (!src.present()) {
+				if (destAbortResult == AbortState::COMPLETED) {
+					printf("The movement has already completed.\n");
+				} else {
+					printf("The movement has been rolled back on the destination cluster.\n");
+				}
+			}
 		}
-		if (dest.present() && destinationPrefix.present()) {
-			wait(abortDBMove(dest.get(), destinationPrefix.get(), MovementLocation::DEST));
+		destinationAbortError = false;
+
+		if (src.present()) {
+			ASSERT(sourcePrefix.present());
+			AbortState tempSrcAbortResult = wait(abortDBMove(src.get(),
```

Comment:
It doesn't look like we are doing the temporary assignment trick here, so we can name this accordingly.

```suggestion
			AbortState srcAbortResult = wait(abortDBMove(src.get(),
```